docs/merkletree/treetex/main.go

```diff
@@ -50,6 +50,16 @@ const (
 \definecolor{perfect}{rgb}{1,0.9,0.5}
 \definecolor{target}{rgb}{0.5,0.5,0.9}
 \definecolor{target_path}{rgb}{0.7,0.7,0.9}
+\definecolor{mega}{rgb}{0.9,0.9,0.9}
+
+\forestset{
```

Comment:
Could you add a comment on what this thing does? :)

Reply:
Whoops, that's a message from the future - you'll see that again in another PR but with a comment attached :)